specification/inference/post_eis_chat_completion/PostEisChatCompletionRequest.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import type { Request as RequestChatCompletion } from '../chat_completion_unified/UnifiedRequest'
+import { Id } from '@_types/common'
+
+
+export type OmittedChatCompletion = Omit<RequestChatCompletion, 'urls' | 'path_parts'>;
```

Comment:
Doesn't look like this is going to work, so either copying the whole `UnifiedRequest` or creating an interface that contains the fields that are shared between `UnifiedRequest` and this file excluding `urls` and `path_parts`.